mapping:
  associationThreshold: 20.0        # probability above which an observation is considered to be known landmark
  associationRange: 10.0
  startingHeadingDeviation: 10.0    # uncertainty associted to the starting heading
  observationIncrement: 1           # when a landmark is observed increments this value
  loopClosureFactor: 0.5            # percentage of the landmarks (missed + seen) that returned
  loopClosureRange: 9
  deleteLandmarkThreshold: 30       #counter of observations limit where a landmark has to be seen

localization:
  initialPosition:
    x: -2.0
    y: 0.0
    theta: 0.087

noise:
  obs:
    d: 0.4                  # range uncertainty in meters
    theta: 3.5              # bearing uncertainty in degrees
  odom:
    x: 0.1                # x uncertainty in meters
    y: 0.1                # y uncertainty in meters
    theta: 0.1             # yaw uncertainty in meters

mapName: "trackdrive"   
daMethod: "ML" 